content/pt-br/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,182 @@
+---
+title: EndpointSlices
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+
+_EndpointSlices_ entregam uma maneira simples de rastrear endpoints de rede através de um cluster Kubernetes. Eles oferecem uma alternativa mais escalável e extensa aos Endpoints.
+<!-- body -->
+
+## Motivação
+
+A Endpoints API oferece uma maneira simples e direta de rastremento de endpoints de rede no Kubernetes. Infelizmente à medida que os clusters Kubernetes e os {{< glossary_tooltip text="Serviços" term_id="service" >}} vem crescido para lidar e mandar mais tráfego para mais Pods do backend, limitações da API original se tornaram mais visíveis.
+Mais notavelmente, estas limitações incluíam desafios com escalar para um númerio maior de endpoints de rede.
+
+Já que todos os endpoints de rede para um serviço são armazenadas em recursos singulares de Endpoints, esses recursos podem ficar um tanto longos. Isso afeta a performance dos componentes do Kubernetes (principalmente a camada de gerenciamento principal) e resultam em quantidades significantes de tráfico na rede e processamento quando os Endpoints são alterados. EndpointSlices ajudam você a mitigar esses problema, além de oferecerem um recurso extenso para adicionar novas funcionalidades como roteamento topológico.
```

Comment:
Maybe change "singulares" to "individuais"? I also think "grandes" would be better than "longos" in this context.

Reply:
This whole paragraph is also written in past tense in the original documentation. Because it is reflecting on something that happened in the past, I think it would be nice to keep the past tense in the Portuguese version as well.